crates/re_log_types/src/data_cell.rs

```diff
@@ -582,9 +578,9 @@ fn data_cell_sizes() {
         );
         cell.compute_size_bytes();
 
-        // 56 bytes for the inner & outer cells + 3x u64s
-        assert_eq!(144, cell.size_bytes());
-        assert_eq!(144, cell.size_bytes());
+        // zero-sized + 3x u64s
+        assert_eq!(136, cell.heap_size_bytes());
+        assert_eq!(136, cell.heap_size_bytes());
```

Comment:
An empty cell of type `InstanceKey` requires 112 bytes of heap space (see test case just above this one) because of the fixed overhead of the datatype's schema, `ComponentName`, etc.

With 3 `InstanceKey`s in the cell, i.e. 3x`u64` that's 112+24=136 bytes of heap space.